### ISSUE TITLE
Creation of saved template for repeated use.

### ISSUE THREAD

**Issue by octopush-it** [vishnuraghavb/EnBizCard#39]
Awesome tool - thanks!  One of the things preventing us from using this is that each card needs to be created by scratch instead of there being a way to templatize a certain layout.  Ideally, a person could configure a bunch of "standard" options like Logo / Company / Theme / Secondary Actions - and then just change the contact details themselves.  This would make creating multiple vCards for many users easier and allow you to leave the tool and come back without needing to reset everything all over again.

Perhaps saving the template as a file, then putting that file inside the working directory?

**Comment by nnhhuu** [comment]
Awesome! It would be perfect if we could save the created cards for later modification.

**Comment by hockeymikey** [comment]
@nnhhuu Agreed. I think being able to save designs would fit both of these. Not sure what is easiest saving to say a json in a subfolder or sqlite? Have a drop down for saved designs, selecting one will load it and there's a save button and a name at the top and save button. If it already exists then a popup asking if you want to overwrite it.
